packages/components/pie-notification/test/visual/pie-notification.spec.ts

```diff
@@ -16,6 +16,7 @@ import { IconHeartFilled } from '@justeattakeaway/pie-icons-webc';
 import {
     variants,
     headingLevels,
+    positions,
```

Comment:
@thejfreitas one thing I have noticed is we don't have any visual tests that set the component to right to left. 

Please could you take a look at the [pie-input visual tests](https://github.com/justeattakeaway/pie/blob/4fca85052c6d9a334346128c870ef13643888a19/packages/components/pie-input/test/visual/pie-input.spec.ts#L16) to see an example of how we can set the page to RTL in order to test it?

Reply:
I believe this prop is not related to reading direction as well don't interfere on the reading experience, however I took the opportunity to provide a visual test for this case scenario. As well I didn't provided a RTL test for each prop because I didn't wanted to double the amount of visual tests since Percy is kinda limited of the number of screenshots. If you think that this component deserves more RTL visual tests I believe that a new ticket would fit better for this purpose. :) 

<img width="1665" alt="Screenshot 2024-04-26 at 11 55 49 AM" src="https://github.com/justeattakeaway/pie/assets/7768202/0831a467-42b7-4467-b050-54308b0d2297">